pkgs/tools/virtualization/awless/default.nix

```diff
@@ -16,7 +16,7 @@ buildGoPackage rec {
   meta = with lib; {
     homepage = "https://github.com/wallix/awless/";
     description = "A Mighty CLI for AWS";
-    platforms = with platforms; linux ++ darwin;
+    platforms = with platforms; [ "i686-linux" "x86_64-linux" ]++ darwin;
```

Comment:
Please use meta.broken instead.

Reply:
Good point -- this was suggested in the larger #173671 PR which marks various packages as broken.